examples/dreamcast/cd/sound-dma/sound-dma.cpp

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
```

Comment:
Right here at the top we're required to add the standard header that has
1. Example Name
2. Author name + year copyright
3. Description of the example